configure.php

```diff
@@ -14,12 +14,19 @@
 	die( 'Not supported in Windows. 🪟' );
 }
 
-function ask( string $question, string $default = '' ): string {
+function ask( string $question, string $default = '', bool $allow_empty = true ): string {
 	$answer = readline(
 		$question . ( $default ? " [{$default}]" : '' ) . ': '
 	);
 
-	return $answer ?: $default;
+	$value = $answer ?: $default;
+
+	if ( ! $allow_empty && empty( $value ) ) {
+		echo "This value can't be empty." . PHP_EOL;
+		return ask( $question, $default, $allow_empty );
```

Comment:
Nice. I like this better than bailing entirely. There are a lot of prompts and it would be frustrating to start over if it bailed.

Reply:
💯 I would be so mad.